marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/entitySearch/getSavedQueries.sjs

```diff
@@ -15,7 +15,7 @@
  */
 'use strict';
 
-xdmp.securityAssert("http://marklogic.com/data-hub/hub-central/privileges/save-entity-query", "execute");
+xdmp.securityAssert("http://marklogic.com/data-hub/privileges/read-entity-model", "execute");
```

Comment:
Why is this read-entity-model instead of save-entity-query? I think a comment would be helpful here, as it's unexpected that this isn't save-entity-query.

Reply:
@rjrudin Reverted it back to normal. I misunderstood the scenario.